config/rbac/clusterwide/role.yaml

```diff
@@ -167,7 +167,7 @@ rules:
 - apiGroups:
   - atlas.mongodb.com
   resources:
-  - atlasfederatedauths
+  - atlasfederatedauths 
```

Comment:
this spurious space can be removed right?

Reply:
Good point, fixed